util/libcrypto.num

```diff
@@ -5465,3 +5465,6 @@ EVP_PKEY_auth_encapsulate_init          ?	3_1_0	EXIST::FUNCTION:
 EVP_PKEY_auth_decapsulate_init          ?	3_1_0	EXIST::FUNCTION:
 PKCS12_SAFEBAG_set0_attrs               ?	3_1_0	EXIST::FUNCTION:
 PKCS12_create_ex2                       ?	3_1_0	EXIST::FUNCTION:
+OSSL_get_thread_support_flags           ?	3_1_0	EXIST::FUNCTION:
+OSSL_set_max_threads                    ?	3_1_0	EXIST::FUNCTION:
```

Comment:
We never seem to call this in the test suite with a value greater than 1 which seems like an omission.